src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj

```diff
@@ -42,9 +42,9 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
-		<PackageReference Include="Umbraco.Cms.Web.BackOffice" Version="[13.0.0,14.0.0)" />
-		<PackageReference Include="Umbraco.Cms.Web.Common" Version="[13.0.0,14.0.0)" />
-		<PackageReference Include="Umbraco.Cms.Web.Website" Version="[13.0.0,14.0.0)" />
+		<PackageReference Include="Umbraco.Cms.Web.BackOffice" Version="[13.2.0,14.0.0)" />
```

Comment:
Please revert lines 45-47 back to `"[13.0.0,14.0.0)"` so we provide the largest support for v13 sites

Reply:
as above